core/lexicon/en/file.inc.php

```diff
@@ -67,6 +67,7 @@
 $_lang['file_folder_err_rename_protected'] = 'Renaming the protected system directory is not permitted.';
 $_lang['file_folder_err_remove'] = 'An error occurred while trying to delete the directory.';
 $_lang['file_folder_err_remove_protected'] = 'Deleting the protected system directory is not permitted.';
+$_lang['file_folder_err_update'] = 'An unknown error occurred while trying to update file the directory.';
```

Comment:
Let's do it the same as for other places, for example when an error happens during deleting
"An error occurred while trying to delete the directory."

Reply:
@alroniks done